napari/layers/image/image.py

```diff
@@ -489,7 +489,18 @@ def _extent_data(self) -> np.ndarray:
         extent_data : array, shape (2, D)
         """
         shape = self.level_shapes[0]
-        return np.vstack([np.zeros(len(shape)), shape])
+        return np.vstack([np.zeros(len(shape)), shape - 1])
+
+    @property
+    def _extent_data_augmented(self) -> np.ndarray:
+        """Extent of layer in data coordinates.
```

Comment:
How is this different from `_extent_data_augmented`? Could we just rely on the base layer's docstring to describe what these are to avoid a doc maintenance burden?